### PR TITLE
Micu/code cleanup

### PR DESCRIPTION
**Pull request by micuintus**

### CODE REVIEW

File: qml/pages/VenueMapPage.qml [hunk 1]
```diff
@@ -48,7 +48,7 @@ BVApp.Page {
 
             onClicked: {
                 var query = venueCoordinate.latitude + "," + venueCoordinate.longitude
-                if (BVApp.Theme.isIos || BVApp.Theme.isOSX) {
+                if (BVApp.Platform.isIos || BVApp.Platform.isMacOS) {
                     Qt.openUrlExternally("https://maps.apple.com/?q=" + query)
```
Comment (micuintus) [inline qml/pages/VenueMapPage.qml]: We could even think about just using the Apple link for all cases as apple redirects to Google for other platforms, just try on GNU/Linux  ;)

Comment (jmastr) [inline qml/pages/VenueMapPage.qml]: There is a possibility to choose between Apple Maps or Google Maps on iOS, but I have not found out yet how to do it. However I am not happy about the patch: v-play has Theme.isIos, Theme.isOSX, ... and I wanted it to be as close to it as possible... Could you take out that patch or revert it or something?!

**Comment by jmastr** [comment]
PR proposal: Please just remove (force push the last two commits: 75c61d4 and 8595573 to have the differentiation between Apple Maps and Geo. I will look further into it once it is deployed. Second: Either adapt the commit msg of 6fdbd44 in regard to isMacOS or rename isOSX to isMacOS in code. That's your decision. What do you say.

Rather than that, nice clean up! Esp. the RO RO part.

File: qml/pages/VenueMapPage.qml (hunk 1 above)
Comment (micuintus) [inline qml/pages/VenueMapPage.qml]: As I already told you IRL: I totally see your point of staying close with v-play upstream. The main reason for me moving these ```isMyHottestSystem()``` questions to a ```Platform``` object was that I assumed the semantics of our questions to differ a bit from the v-play ones: 

With v-play, iOS and Android are actually themes you can set. Of course, by default on Android the Android theme is selected and on iOS the iOS theme, but you can also switch to the iOS Theme on Android if you like, or...

My assumption was that we are going to be adding a ```isSailfishOS()``` question, which would change the semantics of our questions a bit. 

So much for the explanation why I made this change. In the end, I don't care that much ;).

**Comment by micuintus** [comment]
Done, thanks for the testing!

**Pull request merged**